pages/docs/installation/installation-packages.md

```diff
@@ -32,6 +32,22 @@ Please have a look at the official [AUR wiki page](https://wiki.archlinux.org/in
 
 The community also maintains a [development version](https://aur.archlinux.org/packages/precice-git) and [several other packages](https://aur.archlinux.org/packages?&K=precice).
 
+### Nix / NixOS
```

Comment:
```suggestion
## Nix / NixOS
```

(to put it at the same level as Ubuntu/Arch, not under Arch)

Reply:
good catch;)